server/battleships/gateway/implementation/src/main/java/eu/optas/gateway/implementation/InMemoryGameGateway.java

```diff
@@ -29,6 +30,20 @@ public Game createGame() {
         return game;
     }
 
+    @Override
+    public Game getGame(String id) {
+        for (Game game : gameList)
+            if (game.getId().equals(id))
+                return game;
+        return null;
+    }
+
+    @Override
+    public GameStats getGameStats(String id) {
```

Comment:
jeigu kurtum naują gateway, šitą vėl metodo implementaciją vėl duplikuotum? Neturėtų šitas game stats suformavimas būt pačiam use case?